### ISSUE TITLE
Multiple items created when throwing items away

### ISSUE THREAD

**Issue by internecivusraptus** [00-Evan/shattered-pixel-dungeon-gdx#22]
Steps to reproduce:

1) Select some item
2) Press 'throw'
3) Press simultaneously any of these keys: arrow-left arrow-right arrow-up arrow-down
4) Depending on amount of keys pressed multiple items are thrown away

When trying to pickup they all turn back into one item unless you pick them up and use one by one.

![2016-12-01-224945_505x552_scrot](https://cloud.githubusercontent.com/assets/305981/20812007/a1f7646c-b818-11e6-9764-632f9bed251b.png)
![2016-12-01-225002_507x556_scrot](https://cloud.githubusercontent.com/assets/305981/20812005/a1f6816e-b818-11e6-9d1e-ba7ca3a49c72.png)
![2016-12-01-225031_502x555_scrot](https://cloud.githubusercontent.com/assets/305981/20812006/a1f73474-b818-11e6-975e-bd1cd4edd1fc.png)

**Comment by 00-Evan** [comment]
Hmm, interesting. This looks like an existing issue from previous versions as well.

Will fix, thanks.

**Comment by 00-Evan** [comment]
Fixed in commit 3a97545549a0731794b85fdf360c401b13b354f9

**Comment by internecivusraptus** [comment]
I'm not really sure if what I've encountered is related to this bug, but.

I was playing as low-level sniper, encountered piranhas room, dropped a stylus on a door, stepped back and started vigorously clicking 'q', thus launching a boomerang. Once one of them were dead, I noticed that boomerang was lying where the piranha was. I picked it and there were more than one boomerang. Same was with the other piranhas.
